person.rb

```diff
@@ -0,0 +1,43 @@
+class Person
+  # Constructor
+  def initialize(id:, age:, name: 'Unknown', parent_permission: true)
+    # instance variables
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Get method
+  attr_reader :id
+
+  # Get method
+  attr_reader :name
+
+  # Get method
+  attr_reader :age
+
+  # Set method
+  def name=(_name)
+    @name
+  end
+
+  # Set method
+  def age=(_age)
+    @age
+  end
+
+  private
+
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    @age > 18
+  end
+  # rubocop:enable Naming/PredicateName
+
+  public
+
+  def can_use_services?
+    @parent_permission == true || @age > 18
+  end
```

Comment:
- [ ] Please avoid disabling linter rules as they help you write code that follows a consistent format and ensure you're applying some of the best practices in your logic, instead, please consider naming your method `of_age`! 🤓 🔧 

- [ ] **[OPTIONAL]** It's a common practice to leave private methods to the end of your class, this way you don't have to explicitly write the `public` keyword as it is the default, so pretty much all methods and variables above the `private` helper are public, and the ones under are private (unless you use another helper, like protected) 🤓 🔧

- [ ] **[OPTIONAL]** Also, if you look closely, you're repeating the condition of `@age > 18` in both your `of_age` method and your `can_use_services`. Instead, it would be great if you use the `of_age` method to determine whether or not the person can use services, something like this:

```rb
def can_use_services?
  @parent_permission == true || of_age
end
```

Remember the DRY principle!! 🤓 🔧